### PR TITLE
Initial TAG Env website into Spanish

### PR DESCRIPTION
**Pull request by sergioarmgpl**
This is the first PR to contribute to continuous, translate the TAG Env website. We are planning to contribute each month for update this website.

### CODE REVIEW

**Comment by sergioarmgpl** [comment]
For the Spanish grammar validation what do you suggest, remove the grammar validation by adding 

Or add another language to the cspell.json? @leonardpahlke

**Comment by leonardpahlke** [comment]
Thank you so much @sergioarmgpl !!

> For the Spanish grammar validation what do you suggest, remove the grammar validation by adding
>
> Or add another language to the cspell.json? @leonardpahlke

Would defer to @guidemetothemoon here. What are your thoughts?

**Comment by sergioarmgpl** [comment]
@guidemetothemoon :)

**Comment by guidemetothemoon** [comment]
I will check language support in cspell and get back to you😊

**Comment by guidemetothemoon** [comment]
@sergioarmgpl apologies for delayed response here, I finally got time to dig properly into this. So cspell supports a bunch of dictionaries that can be imported, including Spanish one. I made it work locally with cspell tool but it fails with cspell GitHub action and I'm currently in a dialogue with maintainers to find a solution for this: https://github.com/streetsidesoftware/cspell-action/issues/181#issuecomment-1962978997
For now you can ignore that this specific step fails and I will follow it up once I hear back from cspell maintainers.
I also fixed most of the broken links flagged by the link check step directly in your branch - hope it's OK☺️

File: website/content/en/about/working-groups.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ TAG Environmental Sustainability has working groups and teams to effectively col
 
 Following working groups operate under the TAG Environmental Sustainability:
 
-| Name                | Scope and Goals            | Meeting Time                          |
+| Name                | Scope and Goals            | Hora de Reunión                          |
```
Comment (guidemetothemoon) [inline website/content/en/about/working-groups.md:11]:
```suggestion
| Name                | Scope and Goals            | Meeting Time                          |
```
It's an English document so it shouldn't be translated.

**Comment by sergioarmgpl** [comment]
We can merge it @guidemetothemoon cc: @leonardpahlke all good by our side as team @shlopezm4 @jossiebk @Dianmz @JuanPabloGarciaMonzon @MRJeff99 @yimmirp

**Comment by leonardpahlke** [comment]
Awesome!! Merged 🙌

**Pull request merged**